cms/templates/widgets/header.html

```diff
@@ -266,7 +266,7 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
           </li>
           % if static.get_value('ALLOW_PUBLIC_ACCOUNT_CREATION', settings.FEATURES.get('ALLOW_PUBLIC_ACCOUNT_CREATION')):
               <li class="nav-item nav-not-signedin-signup">
-                <a class="action action-signup" href="${settings.FRONTEND_REGISTER_URL}?next=${current_url}">${_("Sign Up")}</a>
+                <a class="action action-signup" href="${settings.FRONTEND_REGISTER_URL}?next=${request.build_absolute_uri(settings.LOGIN_URL)}">${_("Sign Up")}</a>
```

Comment:
I think we should change `current_url` instead of just this bit since when I tried signing up from:
![image](https://user-images.githubusercontent.com/64440265/185648348-4187b9ad-867a-4aeb-8044-81de867cbe19.png)
It doesn't trigger the sso.
Also, I think somewhere quote_plus is applied to current_url. Should we do that to this change as well?

Reply:
Thank you, @mariajgrimaldi

I forgot about that button. I'll update the PR ASAP.

Adding quote_plus is not a problem, I'll add it if you think it's needed (I had no issues without it wile testing though)

Reply:
We could add the quote_plus, so it stays the same if you agree

Reply:
Updated the Sign-Up button and wrapped it with quote_plus, please, check